101-cses-multi-role-rdp/azuredeploy.json

```diff
@@ -0,0 +1,180 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "cloudServiceName": {
+      "type": "string",
```

Comment:
add a defaultValue using  uniqueString() - also mention in the description that this must be globally unique